docs/data/date-pickers/getting-started/getting-started.md

```diff
@@ -55,7 +55,7 @@ yarn add @mui/material @mui/styled-engine-sc styled-components
 ## Code setup
```

Comment:
Maybe we can change the title to help with searches and give a more meaningful title with what needs to be set up.

```suggestion
## Code setup - The role of adapters
```

Reply:
How about 

```suggestion
## Setup your date library adapter
```

That way we have both the idea of "date library" and of "adapter" to optimize the search results.

Reply:
I do like this suggestion! 👍